fdbbackup/backup.actor.cpp

```diff
@@ -3838,6 +3863,11 @@ int main(int argc, char* argv[]) {
 			}
 		}
 
+		if (restoreSystemKeys && restoreUserKeys) {
```

Comment:
Should we also check if individual key-ranges are specified as part of the command?